docs/docs/troubleshooting-common-errors.md

```diff
@@ -0,0 +1,187 @@
+
+### Installing plugins for styling results in Generating SSR bundle failed
+
+If you encounter a webpack error that says `Generating SSR bundle failed` after installing a plugin and trying to run `gatsby develop` or `gatsby build`, it's possible you haven't yet installed all the packages you need.
+
+For some plugins like styled-components, emotion, or SASS, it won't be enough to only install the plugin, you also need to install libraries they rely on. The official installation instructions should guide you to install all needed libraries when you install the plugin, but some tutorials or blog posts you find at other sources may not.
+
+Here are some examples of plugins that require you to install more than just the plugin:
+
+- [gatsby-plugin-emotion](/packages/gatsby-plugin-emotion/): `@emotion/core`, and `@emotion/styled`
+- [gatsby-plugin-styled-components](/packages/gatsby-plugin-styled-components/): `styled-components`, and `babel-plugin-styled-components`
+- [gatsby-plugin-sass](/packages/gatsby-plugin-sass/): `node-sass`, or `dart-sass`
+- [gatsby-image](/packages/gatsby-image/): `gatsby-transformer-sharp`, and `gatsby-plugin-sharp`
+
+Rather than packaging up the other dependent libraries alongside these plugins, they can stay smaller in size when they are published and are able to rely on alternative implementations. One example is `gatsby-plugin-sass` that can use either the Node.js or Dart implementations of SASS.
+
+To resolve these errors, identify the packages that haven't been installed, the error message might look like this:
+
+```shell
+...
+success run queries - 0.095s - 8/8 84.63/s
+
+ ERROR #98123  WEBPACK
+
+Generating SSR bundle failed
+
+Can't resolve '@emotion/core' in '/Users/you/tmp/gatsby-site/.cache' // highlight-line
+
+File: .cache/develop-static-entry.js
+```
+
+This error is a result of Gatsby having failing to find `@emotion/core` because `gatsby-plugin-emotion` has been installed and added to the `gatsby-config`, without installing the emotion library. Install it like this:
+
+```shell
+npm install --save @emotion/core
+```
+
+Or replace `@emotion/core` with the name of the library that is missing. With the plugin installed, any necessary libraries installed, and the plugin added to your `gatsby-config`, it should resolve this error.
+
+## Errors in styling
+
+The following errors are related to styles in your site, using CSS, preprocessors, or CSS-in-JSS solutions.
+
+### Inconsistent CSS styles between develop and build using styled-components or emotion
+
+A common problem that trips up users that install and begin to use styled-components or emotion is forgetting to include the related plugin in the config. Because `gatsby develop` doesn't run server-side rendering, the build may look different if the plugin is not included to tell Gatsby to server-side render the styles for the CSS-in-JS solution being used.
```

Comment:
```suggestion
A common problem that trips up users that install and begin to use styled-components or emotion is not including the related plugin in the config. Because `gatsby develop` doesn't run server-side rendering, the build may look different if the plugin is not included to tell Gatsby to server-side render the styles for the CSS-in-JS solution being used.
```
Recommending this edit because "forgetting" feels a bit like blaming the user. Think it's best to phrase as a step not taken for any number of reasons.